airbyte-integrations/connectors/source-paypal-transaction/pyproject.toml

```
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "2.4.0"
+version = "2.4.1"
```

```suggestion
version = "2.5.0"
```

Schema update is a `minor` change